web/src/types/window.d.ts

```diff
@@ -1,6 +1,7 @@
 // use TypeScript's interface declaration merging
 interface Window {
   env: {
+    [x: string]: string
```

Comment:
Is this necessary?

Reply:
Thanks, accidental, fixed in: https://github.com/bcgov/wps/pull/3557/commits/df7c95d01c40578f4652f107f2557febb581bd4c